tools/scripts/README.md

```diff
@@ -0,0 +1,25 @@
+# Scripts
```

Comment:
Let me know if I need to clean anything up the template had it in above ☝️ 

The actual interesting part is below 👇